config/config_test.go

```diff
@@ -522,6 +523,37 @@ func TestConfigEnsurePartialStart(t *testing.T) {
 	test.That(t, invalidAuthConfig.Ensure(false), test.ShouldBeNil)
 }
 
+func TestRemoteValidate(t *testing.T) {
+	t.Run("remote invalid name", func(t *testing.T) {
```

Comment:
this test largely replicates the test `TestComponentValidate`. if we consolidate regexes we can have one test for the name validation regex and then just check that it is being applied to each name